### PR TITLE
Theming css spinner

### PR DESCRIPTION
**Pull request by Gabss405**
adding themeable css spinner

### CODE REVIEW

File: src/components/atoms/Spinner/CustomSpinner/CustomSpinner.tsx [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+    from {
+      transform: rotate(0deg);
+    }
+    to {
+      transform: rotate(360deg);
+    }
+  }
+`;
+
+const CustomSpinner: React.FC<CustomSpinnerProps> = ({ size, styling }) => {
```
Comment (admmasters) [inline src/components/atoms/Spinner/CustomSpinner/CustomSpinner.tsx:81]: This classic: https://backstage.io/docs/architecture-decisions/adrs-adr006

Comment (admmasters) [reply src/components/atoms/Spinner/CustomSpinner/CustomSpinner.tsx:81]: We are going to have to do a bit of refactor of the types in zrc, but shall we do it the "modern" way for new bits of code? Maybe `PropsWithChildren`?

**Review by calvin-fung-zopa — APPROVED** [review]
LGTM!

**Review by admmasters — APPROVED** [review]

**Pull request merged**